### ISSUE TITLE
Very poor performance on custom dataset with partly overlapping instances

### ISSUE THREAD

**Issue by Ma1hias** [open-mmlab/mmdetection#7445]
Hi,

I trained different default model configurations (mask_rcnn, cascade) with different backbones (swin, r50, x101)  for a custom COCO formatted dataset. The dataset contains only one class with up to 100 instances per image, but most of the instances have a very small overlapping region with one or more other instances.

The results are very strange for all tested models because only a fraction of the instances got recognized but the accuracy on the few detected instances is actually very good.

Using Detectron2 I got much better results on the very same dataset, but I would like to use mmdetection.

Do you have any suggestions what could be the reason for this behavior?

Thank you for your time,
Mathias

**Comment by RangiLyu** [comment]
Can you upload your config and provide more details such as how many GPUS you use and the hyperparams you use?

**Comment by RangiLyu** [comment]
The learning rate in this config is for 8 GPUs (total batch size = 16). Please adjust the hyperparameters.

**Comment by Ma1hias** [comment]
You are right, in this case I did not adjust the learning rate.

I also trained the model with the 1/8 the learning rate ans 2 images per gpu and got similar results. 

Do you have any other idea what could be the reason for the poor performance?

**Comment by muneebelahimalik** [comment]
> You are right, in this case I did not adjust the learning rate.
> 
> I also trained the model with the 1/8 the learning rate ans 2 images per gpu and got similar results. The precision is high but the recall is very low.
> 
> Do you have any other idea what could be the reason for the poor performance?

Hey! Did you find any solution for this?
